packages/dds/tree/src/simple-tree/core/index.ts

```diff
@@ -25,17 +25,18 @@ export {
 	inPrototypeChain,
 } from "./treeNode.js";
 export { NodeKind, isTreeNodeSchemaClass } from "./treeNodeSchema.js";
-export type {
-	TreeNodeSchema,
-	TreeNodeSchemaClass,
-	TreeNodeSchemaNonClass,
-	TreeNodeSchemaCore,
-	TreeNodeSchemaBoth,
-	NodeSchemaMetadata,
-	TreeLeafValue,
-	InsertableTypedNode,
-	NodeBuilderData,
-	NodeFromSchema,
+export {
+	type TreeNodeSchema,
+	type TreeNodeSchemaClass,
+	type TreeNodeSchemaNonClass,
+	type TreeNodeSchemaCore,
+	type TreeNodeSchemaBoth,
+	type NodeSchemaMetadata,
+	type TreeLeafValue,
+	type InsertableTypedNode,
+	type NodeBuilderData,
+	type NodeFromSchema,
+	asTreeNodeSchemaCorePrivate,
```

Comment:
add this non-type export to the none type ones on line 27 rather than modifying this import.